diff -urN howl-0.9.10.orig/howl.pc.in howl-0.9.10/howl.pc.in
--- howl-0.9.10.orig/howl.pc.in	2005-03-20 16:47:30.018681568 +0100
+++ howl-0.9.10/howl.pc.in	2005-03-20 16:55:45.821308112 +0100
@@ -3,10 +3,14 @@
 # For more info, see <http://www.freedesktop.org/software/pkgconfig/>
 #
 
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
 Name: Howl
 Description: Howl Rendezvous/mDNS-SD Library from Porchdog Software (http://www.porchdogsoft.com)
 Version: @VERSION@
 Requires:
-Libs: -L@prefix@/lib -lhowl @PLATFORM_LIBS@
-Cflags: -I@prefix@/include/howl
-
+Libs: -L${libdir} -lhowl @PLATFORM_LIBS@
+Cflags: -I${includedir}/howl-@VERSION@
