Index: include/salt/vtypes.h
===================================================================
RCS file: /cvsroot/howl/howl/include/salt/vtypes.h,v
retrieving revision 1.13
diff -u -p -r1.13 vtypes.h
--- include/salt/vtypes.h	17 Apr 2004 21:51:40 -0000	1.13
+++ include/salt/vtypes.h	22 Jun 2004 14:15:23 -0000
@@ -56,7 +56,7 @@ typedef unsigned char		sw_uchar;				
 typedef short					sw_short;
 typedef unsigned short		sw_ushort;
 typedef long					sw_long;
-typedef unsigned long		sw_ulong;
+typedef unsigned int		sw_ulong;
 typedef unsigned char	*	sw_octets;
 typedef char				*	sw_string;
 #if !defined(__VXWORKS__) || defined(__cplusplus)
