Index: src/lib/howl/Posix/posix_interface.c
===================================================================
RCS file: /cvsroot/howl/howl/src/lib/howl/Posix/posix_interface.c,v
retrieving revision 1.3
diff -u -p -r1.3 posix_interface.c
--- src/lib/howl/Posix/posix_interface.c	17 Apr 2004 21:51:42 -0000	1.3
+++ src/lib/howl/Posix/posix_interface.c	23 Jun 2004 11:03:57 -0000
@@ -94,14 +94,7 @@ get_sa_len(struct ifreq * ifr)
 int 
 get_sa_len(struct ifreq * ifr)
 {
-#ifdef  IPV6
-	if (ifr->ifr_addr.sa_family == AF_INET6)
-	{
-		return sizeof(struct sockaddr_in6);
-	}
-#endif
-	/* AF_INET */
-	return sizeof(struct sockaddr);
+	return sizeof(ifr->ifr_ifru);
 }
 #endif  
 
