Index: include/salt/vtypes.h
===================================================================
RCS file: /cvsroot/howl/howl/include/salt/vtypes.h,v
retrieving revision 1.13
diff -u -p -r1.13 vtypes.h
--- include/salt/vtypes.h	17 Apr 2004 21:51:40 -0000	1.13
+++ include/salt/vtypes.h	23 Jun 2004 07:22:03 -0000
@@ -119,15 +120,13 @@ typedef _int64					sw_verylong;
 
 #elif defined(__linux__)
 
-#	if defined(i386)
+#include <endian.h>
+  
+#if __BYTE_ORDER == __LITTLE_ENDIAN
 #		define SW_ENDIAN		1
-#	elif defined(__sparc__)
+#elif __BYTE_ORDER == __BIG_ENDIAN
 #		define SW_ENDIAN		0
-#	elif defined(powerpc)
-#		define SW_ENDIAN		0
-#	else
-#		error "CPU unknown"
-#	endif
+#endif
 
 #elif defined(__VXWORKS__)
 
