diff -urN howl-0.9.10.orig/src/autoipd/Makefile.am howl-0.9.10/src/autoipd/Makefile.am
--- howl-0.9.10.orig/src/autoipd/Makefile.am	2005-03-20 16:47:30.059675336 +0100
+++ howl-0.9.10/src/autoipd/Makefile.am	2005-03-20 16:52:06.737613896 +0100
@@ -1,7 +1,5 @@
-SUBDIRS						=	Posix Linux
-INCLUDES						=	-I$(top_srcdir)/include/
-LDADD							=	$(AUTOIPD_EXTRA_OBJECTS) $(top_builddir)/src/lib/howl/libhowl.la $(PLATFORM_LIBS)
-bin_PROGRAMS				=	autoipd
-autoipd_SOURCES			=	autoip.c autoip.h
-EXTRA_autoipd_SOURCES	=	Linux/linux_autoip.c Posix/posix_main.c
-autoipd_DEPENDENCIES		=	linux_autoip.o posix_main.o
+SUBDIRS		= Posix Linux
+INCLUDES	= -I$(top_srcdir)/include/
+LDADD		= $(top_srcdir)/src/lib/howl/libhowl.la $(PLATFORM_LIBS)
+bin_PROGRAMS	= autoipd
+autoipd_SOURCES	= autoip.c autoip.h Linux/linux_autoip.c Posix/posix_main.c
