Index: configure.in
===================================================================
RCS file: /cvsroot/howl/howl/configure.in,v
retrieving revision 1.48
diff -u -p -r1.48 configure.in
--- configure.in	21 Apr 2004 06:13:31 -0000	1.48
+++ configure.in	22 Jun 2004 13:56:12 -0000
@@ -52,6 +53,7 @@ AC_SUBST([HOWL_LIB_OBJECTS])
 AC_SUBST([AUTOIPD_EXTRA_OBJECTS])
 AC_SUBST([PLATFORM_LIBS])
 AC_SUBST([HOWL_MAN_PAGES])
+AC_CHECK_HEADERS([stdint.h])
 AC_OUTPUT(
 				Makefile										\
 				include/Makefile							\
Index: src/lib/howl/NotOSX/DNSServices.h
===================================================================
RCS file: /cvsroot/howl/howl/src/lib/howl/NotOSX/DNSServices.h,v
retrieving revision 1.1
diff -u -p -r1.1 DNSServices.h
--- src/lib/howl/NotOSX/DNSServices.h	1 Mar 2004 21:14:47 -0000	1.1
+++ src/lib/howl/NotOSX/DNSServices.h	22 Jun 2004 13:56:13 -0000
@@ -98,6 +98,10 @@ DNS Services for Windows
 #include	<stddef.h>
 #include <salt/salt.h>
 
+#ifdef HAVE_STDINT_H
+#include <stdint.h>
+#endif
+
 #ifdef	__cplusplus
 	extern "C" {
 #endif
@@ -224,7 +228,11 @@ DNS Services for Windows
 	@abstract	8-bit unsigned data type.
 */
 
+#ifdef HAVE_STDINT_H
+typedef uint8_t			DNSUInt8;
+#else
 typedef unsigned char		DNSUInt8;
+#endif
 
 dns_check_compile_time( sizeof( DNSUInt8 ) == 1 );
 
@@ -234,8 +242,12 @@ dns_check_compile_time( sizeof( DNSUInt8
 	@abstract	16-bit unsigned data type.
 */
 
+#ifdef HAVE_STDINT_H
+typedef uint16_t		DNSUInt16;
+#else
 typedef unsigned short		DNSUInt16;
-
+#endif
+	  
 dns_check_compile_time( sizeof( DNSUInt16 ) == 2 );
 
 //---------------------------------------------------------------------------------------------------------------------------
@@ -244,7 +256,11 @@ dns_check_compile_time( sizeof( DNSUInt1
 	@abstract	32-bit unsigned data type.
 */
 
+#ifdef HAVE_STDINT_H
+typedef uint32_t		DNSUInt32;
+#else
 typedef unsigned long		DNSUInt32;
+#endif
 
 dns_check_compile_time( sizeof( DNSUInt32 ) == 4 );
 
@@ -254,7 +270,11 @@ dns_check_compile_time( sizeof( DNSUInt3
 	@abstract	32-bit signed data type.
 */
 
+#ifdef HAVE_STDINT_H
+typedef int32_t			DNSSInt32;
+#else
 typedef signed long		DNSSInt32;
+#endif
 
 dns_check_compile_time( sizeof( DNSSInt32 ) == 4 );
 
